source/tutorial/subroutes.md

```diff
@@ -205,7 +205,7 @@ export default function() {
         title: "Downtown Charm",
         owner: "Violet Beauregarde",
         city: "Portland",
-        "property-type": "Apartment",
+        Category: "Apartment",
```

Comment:
Lowercase category here, right?

Reply:
Thanks for catching this. Yes. Lowercase

Reply:
Hello @EdgarAllenPoe, can you push the correction to make `Category` lowercase? Then we'll get this merged. Thank you!